### DISCUSSION TITLE
Image conversion - HEIC to Jpg

### DISCUSSION THREAD

**Discussion by Elanchezhiyan-P** [artehe/Netimobiledevice#21] [Ideas]
Hi @artehe , 

Do we have any inbuilt or default function for converting the image format from HEIC to jpg or png? I have tried using [Imagick](https://www.nuget.org/packages/Magick.NET.Core) but it's size is more. Do you have any idea on this process?

**Comment by artehe** [comment]
No, we don't.

No idea but a quick google threw this up: https://stackoverflow.com/questions/56939187/converting-heic-to-jpeg-using-imagick-in-c-sharp

**Comment by Elanchezhiyan-P** [comment]
Yes, I too have noticed and it works fine. But it's size is too large. That's why preferring for other packages. Let me know if you know any other packages.

**Comment by artehe** [comment]
I do not know I'm afraid, never done much with image conversion

**Reply by Elanchezhiyan-P** [reply]
Thanks for the help, @artehe . I have used the [FileOnQ.Imaging.Heif](https://www.nuget.org/packages/FileOnQ.Imaging.Heif) package to convert the images from HEIC format to JPG. It works fine as expected.
